.github/workflows/connector-tests.yml

```diff
@@ -131,15 +133,18 @@ jobs:
         run: |
           cd airbyte
           make tools.airbyte-ci-dev.install
-          airbyte-ci-dev connectors \
+          airbyte-ci-dev \
+          --ci-report-bucket-name=airbyte-ci-reports-multi \
```

Comment:
Specifying a bucket name is now sufficient to tell `airbyte-ci` to upload the reports to GCS.